src/test/java/org/broadinstitute/hellbender/tools/genomicsdb/GenomicsDBImportIntegrationTest.java

```diff
@@ -42,18 +43,23 @@
     private static final String HG_00096 = largeFileTestDir + "gvcfs/HG00096.g.vcf.gz";
     private static final String HG_00268 = largeFileTestDir + "gvcfs/HG00268.g.vcf.gz";
     private static final String NA_19625 = largeFileTestDir + "gvcfs/NA19625.g.vcf.gz";
+    private static final String NA_12878_PHASED = largeFileTestDir + "NA12878.phasedData.Chr20.vcf";
```

Comment:
Can you note that this is a VCF with HaplotypeCaller PGT:PID phasing info, which isn't really what you wanted for phasing, but it's nice to test it anyway.

Reply:
Oh yeah, thats right. I will mark it. I ended up adding the artifical phased data  file for this purpose.